evalplus/eval/__init__.py

```diff
@@ -228,6 +229,7 @@ def evaluate_files(
     ref_time: List[float],
     fast_check: bool = False,
     min_time_limit: float = 0.1,
+    gt_timelimit_multiplier: float = 2.0,

```

Comment:
How about go gt_time_limit_factor 

Separating time and limit to be consistent with min_time_limit. Use factor for being short.